daemon-scheduler/internal/features/steps/step_defns.go

```diff
@@ -655,7 +655,7 @@ func doSomething(ttl time.Duration, tickTime time.Duration, fn func() (bool, err
 }
 
 func filterTasksByStatusRunning(cluster *string, taskARNs []*string, ecsWrapper wrappers.ECSWrapper) []*string {
-	runningTasks := make([]*string, len(taskARNs))
+	runningTasks := make([]*string, 0)
```

Comment:
Changed. Also added an additional check to make sure that ECS does not return more than `len(taskARNs)`.